colin/core/ruleset/ruleset.py

```diff
@@ -20,45 +20,45 @@
 
 from six import iteritems
 
-from ..exceptions import ColinConfigException
-from ..constant import CONFIG_DIRECTORY, JSON
+from ..exceptions import ColinRulesetException
+from ..constant import RULESET_DIRECTORY, JSON
 from ..loader import load_check_implementation
 from ..target import is_compatible
 
 logger = logging.getLogger(__name__)
 
 
-class Config(object):
+class Ruleset(object):
 
-    def __init__(self, config_name=None, config_file=None):
+    def __init__(self, ruleset_name=None, ruleset_file=None):
         """
-        Load config for colin.
+        Load ruleset for colin.
 
-        :param config_name: str (name of the config file (without .json), default is "default"
-        :param config_file: fileobj
+        :param ruleset_name: str (name of the ruleset file (without .json), default is "default"
+        :param ruleset_file: fileobj
```

Comment:
`ruleset_file` looks more like a str with a path to me. Am I missing something ?

Reply:
No, it's really the file object. (click checks the permission and gives us the file object)

Reply:
Good point. Corrected.